src/stories/_exec/__init__.py

```diff
@@ -0,0 +1,26 @@
+from ..exceptions import StoryDefinitionError
+from . import function, coroutine
+
+try:
+    from inspect import iscoroutinefunction
+except SyntaxError:
```

Comment:
Does python 2 raize `SyntaxError` on import of the simple predicate function? 🤯
I thought it should be an `ImportError`.